.travis.yml

```diff
@@ -11,6 +11,7 @@ env:
   - PIP_DIR="$HOME/virtualenv"
   - GATK_PATH="$CACHE_DIR"
   - PYTHONIOENCODING=UTF8
+  - PYTEST_ADDOPTS="-rsxX -n 2 --durations=25 --fixture-durations=10 --junit-xml=pytest.xml --cov-report= --cov broad_utils --cov illumina --cov assembly --cov interhost --cov intrahost --cov metagenomics --cov ncbi --cov read_utils --cov reports --cov taxon_filter --cov tools --cov util --cov file_utils"
```

Comment:
I had moved these travis env variables to the parts of the build matrix that actually use py.test... there are a bunch of other points in the build matrix that have nothing to do with pytest.

Reply:
Right; on the other hand, there are parts that don't use GATK_PATH either.   There's a downside to having copies of the same complex line in several places -- the copies have to be kept in sync.
But if having PYTEST_ADDOPTS defined in jobs that don't use it might cause problems, I'll revert this change.